htdocs/comm/action/class/cactioncomm.class.php

```diff
@@ -341,5 +341,6 @@ public function getNomUrl($withpicto = 0)
 		if ($transcode != "Action".$this->code) {
 			return $transcode;
 		}
+		return -1;
```

Comment:
So comment doc say "@return string Label of action type"
So you must choose between making a return '';
or a return -1 and you modify the doc to say  "* @return int|string Label of action type"